shopinvader_dropoff_site_edition/services/cart.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.addons.component.core import Component
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class CartService(Component):
+    _inherit = "shopinvader.cart.service"
+
+    def set_public_dropoff_site(self, **params):
```

Comment:
@sebastienbeau Why do you put this logic on the cart service? It should go into `shopinvader.delivery.pickup.service`Moreover, dropoff must be renamed into pickup.

Reply:
it's done now